processor/src/main/java/com/workday/autoparse/json/codegen/JsonObjectParserTableGenerator.java

```diff
@@ -71,7 +72,11 @@ public void generateParserMap() throws IOException {
                                                                          GeneratedClassNames
                                                                                  .CLASS_JSON_OBJECT_PARSER_TABLE);
 
-        JavaFileObject sourceFile = processingEnv.getFiler().createSourceFile(qualifiedClassName);
+        Map<String, TypeElement> parserMap = new HashMap<>();
+        parserMap.putAll(discrimValueToClassRequiringGeneratedParserMap);
+        parserMap.putAll(discrimValueToClassWithCustomParserMap);
```

Comment:
what's the difference b/w `discrimValueToClassRequiringGeneratedParserMap` and `discrimValueToClassWithCustomParserMap` ?

Reply:
Autoparse JSON supports optional custom parsers, which are added to `discrimValueToClassWithCustomParserMap`. If they're not included here the processor should miss these custom parsers.